### ISSUE TITLE
Skip Content link is not visible when the side menu is open

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#130]
Screengrab/videos (insert file names here)
2020 08 18, Skip content link is not visible when the side menu is open.mp4

Link
https://d8-stg.boston.gov/

Node # (if known)
21

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.6 > Chrome 84

Trusted Tester Test
Skip Repetitive Content

Brief Description
Skip Content link is not visible when the side menu is open

Steps to reproduce
1)	00:03 – Tab over to the skip content link to the side menu
2)	00:06 – Press Shift and tab to tab backwards to the Skip content link

Desired Result
•	The skip content link is visible on the page when the side menu is open

Actual Result
•	The skip content link is not visible on the page when the side menu is open
[2020 08 18, Skip content link is not visible when the side menu is open.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5098888/2020.08.18.Skip.content.link.is.not.visible.when.the.side.menu.is.open.mp4.zip)

**Comment by subaha-cob** [comment]
Testing at: https://d8-uat.boston.gov/

**Comment by david-iterators** [comment]
Duplicate of Issue #138, so closing this issue.
